### PR TITLE
Check provisioning status before provisioning.

### PR DESCRIPTION
**Pull request by parameshbabu**
Check provisioning status before applying provisioning. 
Re-provisioning not supported yet.

### CODE REVIEW

File: eflowautodeploy/eflowAutoDeploy.ps1 [hunk 1]
```diff
@@ -493,6 +493,7 @@ function Test-EadEflowInstall {
     Write-Host "AzureEFLOW Module:$version"
     $svc = Get-Service wssdagent
     if ($svc.Status -ne 'Running') {
+        Set-Service -Name wssdagent -StartupType Automatic
```
Comment (fcabrera23) [inline eflowautodeploy/eflowAutoDeploy.ps1:496]: Why are we adding the StartupType Automatic?

File: eflowautodeploy/eflowAutoDeploy.ps1 [hunk 2]
```diff
@@ -917,6 +934,10 @@ function Start-EadWorkflow {
         if (!(Test-EadEflowVMSwitch -Create)) { return $false } #create switch if specified
         # We are here.. all is good so far. Validate and deploy eflow
         if (!(Invoke-EadEflowDeploy)) { return $false }
+    }
+    if (Test-EadEflowVMProvision) {
+        Write-Host "EFLOW VM is already provisioned." -ForegroundColor Yellow
```
Comment (fcabrera23) [inline eflowautodeploy/eflowAutoDeploy.ps1:939]: I would change the message - It's not really provisioned (maybe it failed), it's just the configuration was applied, or "there was an attempt to provision"

Comment (parameshbabu) [reply eflowautodeploy/eflowAutoDeploy.ps1]: removed this completely.

File: eflowautodeploy/eflowAutoDeploy.ps1 (hunk 1 above)
Comment (parameshbabu) [reply eflowautodeploy/eflowAutoDeploy.ps1]: removed this completely

**Pull request merged**